win32/src/winapi/user32/window.rs

```diff
@@ -909,6 +909,15 @@ pub fn GetWindowRect(machine: &mut Machine, hWnd: HWND, lpRect: Option<&mut RECT
     true
 }
 
+#[win32_derive::dllexport]
+pub fn GetWindowPlacement(
+    _machine: &mut Machine,
+    hWnd: HWND,
+    lpwndpl: Option<&mut WINDOWPLACEMENT>,
```

Comment:
This PR was missing the definition of WINDOWPLACEMENT :)

Reply:
Sorry, I missed it when extracting this from #67 🙈